tests/unit/helpers.js

```diff
@@ -1322,6 +1333,16 @@ export const createFakeTracking = (overrides = {}) => {
   };
 };
 
+export const createFakeTrackingWithJest = (overrides = {}) => {
```

Comment:
`createFakeTracking` is used by other tests that have not been converted yet, so I created this version for the updated tests. Once everything that uses this has been converted we can delete `createFakeTracking` and rename `createFakeTrackingWithJest` to `createFakeTracking`.

Reply:
Same answer :-)